journal/modernized/37-chapter-34.adoc

```diff
@@ -1637,6 +1638,7 @@ accidentally and without any foundation in fact;
 but this slender ray of hope was of short duration.
 
 Sixth Month 11th. (First day.)--Since the 30th ultimo my pen
+//Have you seen "ultimo" above before? I think it means "in the months preceding present". Do you want me to change this in the modernized? Sounds like his pen's named ultimo. :P
```

Comment:
lol, that's funny about his pen being named ultimo 🤣 

Yeah, they use "instant" (meaning "this month") and "ultimo" (meaning "last month") a lot, when writing letters or referencing letters, or in journal entries.  More commonly using the abbreviations `inst.` and `ult.`. We batted this around a couple weeks ago and landed that we think in _modernized_ versions we should change them to "of last month" and "of this month", and leave them alone in original. So let's change this to read:

```ASCIIDOC
Sixth Month 11th. (First day.)--Since the 30th of last month my pen
```